### PR TITLE
feat(toolbar): dense toolbar sass

### PR DESCRIPTION
**Pull request by philgetz**
Adding dense toolbars

`<mat-toolbar dense>` wil make toolbars be 48px

`<td-layout dense='mat-toolbars'> ` will propagate to all toolbars contained to height 48px

`<body dense='mat-toolbars'>` will make mat-toolbar 48px globally

### CODE REVIEW

File: src/lib/theme/_overrides.scss [hunk 1]
```diff
@@ -224,6 +224,48 @@ $shape-radius: 8px;
...
+      .mat-toolbar-single-row {
+        height: 48px;
+      }
+    }
+    // Dense toolbars
+    // * Setting this on wil propagate to all mat-toolbars
+    td-layout,
+    td-layout-nav,
+    td-layout-manage-list {
+      &[dense='mat-toolbars'] {
```
Comment (emoralesb05) [inline src/lib/theme/_overrides.scss]: Shouldnt this be just `[dense]`?

File: src/lib/theme/_overrides.scss [hunk 2]
```diff
@@ -224,6 +224,48 @@ $shape-radius: 8px;
       height: 3px;
       border-radius: 3px 3px 0 0;
     }
+
+    // MatToolbars globally 48px
+    // Adding this to body will make all mat-toolbars 48px in height
+    &[dense='mat-toolbars'] {
```
Comment (emoralesb05) [inline src/lib/theme/_overrides.scss]: same with this? just `[dense]`?

**Comment by emoralesb05** [comment]
Now that i think about it.. maybe dense should be an attribute class of layouts to target only things inside the component and it doesnt cascade.

File: src/lib/theme/_overrides.scss (hunk 1 above)
Comment (emoralesb05) [reply src/lib/theme/_overrides.scss]: Now that i think about it.. maybe dense should be an attribute class of layouts to target only things inside the component and it doesnt cascade.

Comment (christianmemije) [reply src/lib/theme/_overrides.scss]: Yeah I agree that this should probably go inside Covalent

Comment (emoralesb05) [reply src/lib/theme/_overrides.scss]: Did a super quick pass here: https://github.com/Teradata/covalent/pull/1663

**Comment by emoralesb05** [comment]
We can remove the SCSS code since this was added into covalent already.

**Pull request closed without merging**